### DISCUSSION TITLE
Where do I go to see which limits have been exceed?

### DISCUSSION THREAD

**Discussion by cossieB** [vercel/community#6694] [Account Issue] [answered]
### Question

I'm on the hobby plan and last month I exceeded my PG limits for the first time ever. However my dashboard says every metric is below its threshold - compute time is very close to its threshold but still below.
![Screenshot 2024-05-01 121719](https://github.com/vercel/community/assets/91845390/2171d57a-cf06-4456-b607-c002b75a1360)

**Comment by amyegan** [comment]
You're on the right page to see database usage metrics. You can also view those metrics on the Usage page of your dashboard. 

Are you experiencing other warnings and restrictions that would indicate your db has been paused, or does the incorrect status badge seem to be the only issue?

**Reply by cossieB** [reply, marked as answer]
Strange. The issue seems to have cleared itself up now 🤣. My database was down for the last week or so. I couldn't even connect with psql.

**Reply by amyegan** [reply]
I'm glad it's working now! Please let me know if you see that problem again
